plugins/fabric8-cdn-support/src/main/java/com/redhat/che/cdn/CdnSupportService.java

```diff
@@ -265,8 +266,7 @@ private PluginMeta getEditorMeta() throws InfrastructureException {
         pluginFQN.getRegistry() == null
             ? pluginRegistry.clone()
             : UriBuilder.fromUri(pluginFQN.getRegistry());
-
-    URI editorURI = uriBuilder.path(pluginFQN.getId()).build();
+    URI editorURI = uriBuilder.path(pluginFQN.getId()).path(META_YAML).build();
```

Comment:
Minor quibble: since we're building the full URL here, we may as well include the plugins part of the path. This isn't too important though.
```suggestion
    URI editorURI = uriBuilder..path("plugins").path(pluginFQN.getId()).path(META_YAML).build();
```

Reply:
@amisevsk good point, but I will probably include this into upcoming PR for beta 5 update